cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -99,7 +99,7 @@ else ()
 
   ExternalProject_Add (snappy_ep
     URL "https://github.com/google/snappy/archive/${SNAPPY_VERSION}.tar.gz"
-    CMAKE_ARGS ${SNAPPY_CMAKE_ARGS}
+    CMAKE_ARGS ${SNAPPY_CMAKE_ARGS} -DSNAPPY_BUILD_TESTS=OFF
```

Comment:
Thank you for suggestion, but why do we need to upgrade when we disable like this?

Reply:
Same question here.

Reply:
I did it because it seems not to have negative impact, but I don't think it's needed to fix the issue. I can change the PR not to include this version bump if you prefer.

Reply:
Thanks for the explanation! Do you mind creating a JIRA for this?

Reply:
Sorry, I don't have a Jira account (and I'd rather not have to create one just for this very simple MR).

Reply:
Understood. Let me do that for you, @douardda .

Reply:
Thanks!